Cargo.toml

```diff
@@ -79,7 +79,7 @@ abi3-py39 = ["abi3-py310", "pyo3-build-config/abi3-py39", "pyo3-ffi/abi3-py39"]
 abi3-py310 = ["abi3", "pyo3-build-config/abi3-py310", "pyo3-ffi/abi3-py310"]
 
 # Automatically generates `python3.dll` import libraries for Windows targets.
-generate-abi3-import-lib = ["pyo3-build-config/python3-dll-a"]
+generate-abi3-import-lib = ["pyo3-build-config/python3-dll-a", "pyo3-ffi/generate-abi3-import-lib"]
```

Comment:
As per https://github.com/PyO3/pyo3/pull/2309/files#r851606558, I think it's good enough to just have the `pyo3-ffi` one here?